docs/docs/40-operator-guide/40-security/40-managing-credentials.md

```diff
@@ -258,37 +258,53 @@ obtain a token that is valid for 12 hours and cached for 10.
 Kargo can be configured to authenticate to Google Artifact Registry repositories
 using Workload Identity Federation.
 
+#### Cluster Configuration
+
 If Kargo locates no `Secret` resources matching a repository URL, and if Kargo
 is deployed within a GKE cluster, it will attempt to use
 [Workload Identity Federation](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
-to authenticate, but this relies upon some external setup. Leveraging this
-option eliminates the need to store credentials in a `Secret` resource.
+to authenticate by default. Leveraging this
```

Comment:
We should probably remove "by default" since there is not, at present, a way to configure WIF to _not_ be used if it's detected as being available.